tests/e2e/lib.sh

```diff
@@ -353,25 +356,39 @@ deploy_optional_e2e_components() {
 
     if [[ "${INSTALL_COMPLIANCE_OPERATOR:-false}" == "true" ]]; then
         install_the_compliance_operator
+    elif [[ "${INSTALL_COMPLIANCE_OPERATOR_FROM_SOURCE:-false}" == "true" ]]; then
+        install_the_compliance_operator_from_source
     else
         info "Skipping the compliance operator install"
     fi
 }
 
 install_the_compliance_operator() {
-    info "Installing the compliance operator"
-
-    # ref: https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-installation.html
-
-    oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/namespace.yaml"
-    oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/operator-group.yaml"
-    oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/subscription.yaml"
-
-    wait_for_object_to_appear openshift-compliance deploy/compliance-operator
+    info "Installing the compliance operator from subscription"
+
+    csv=$(oc get csv -n openshift-compliance -o json | jq ".items[] | select(.metadata.name | test(\"compliance-operator\")).metadata.name")
+    if [[ $csv == "" ]]; then
+        # ref: https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-installation.html
+        oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/namespace.yaml"
+        oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/operator-group.yaml"
+        oc create -f "${ROOT}/tests/e2e/yaml/compliance-operator/subscription.yaml"
+        wait_for_object_to_appear openshift-compliance deploy/compliance-operator
+    else
+        info "Reusing existing compliance operator deployment from $csv subscription"
+    fi
 
     oc get csv -n openshift-compliance
 }
 
+install_the_compliance_operator_from_source() {
```

Comment:
This might not be necessary anymore if we do https://github.com/stackrox/stackrox/pull/9043 - which installs the compliance operator for upstream catalog indexes.

Reply:
Well - maybe. This might not work on GKE if it's dependent on OLM.